### ISSUE TITLE
CM install of Chef failure

### ISSUE THREAD

**Issue by Sauraus** [boxcutter/windows#56]
```
vmware-iso: https://opscode-omnibus-packages.s3.amazonaws.com/windows/2008r2/x86_64/chef-windows-12.4.3.windows.msi:
vmware-iso: ==> Downloading "https://opscode-omnibus-packages.s3.amazonaws.com/windows/2008r2/x86_64/chef-windows-12.4.3.windows.msi" to "C:\Users\vagrant\AppData\Local\Temp\chef\chef-client-latest.msi"
vmware-iso: 2015-12-17 11:04:53 ERROR 404: Not Found.
```

**Comment by dhekimian** [comment]
Reviewing the [Chef Download](https://downloads.chef.io/chef-client/windows/) site between versions 12.4.1 and 12.4.2 they changed the download URL:
```

https://opscode-omnibus-packages.s3.amazonaws.com/windows/2008r2/x86_64/chef-client-12.4.1-1.msi
https://opscode-omnibus-packages.s3.amazonaws.com/windows/2012r2/i386/chef-client-12.4.2-1-x86.msi
```

Looks like `/script/cmtool.bat` needs to be modified to handle the change in URL.

**Comment by daxgames** [comment]
The problem here is cmtool.bat is assuming there is an x64 version of chef client available but that is not always the case.  x64 builds are not available until chef-client 12.7.2.  cmtool.bat needs to be fixed to default to x86 versions of Chef if the CM_VER is less than 12.7.2.

**Comment by daxgames** [comment]
Quick and dirty fix, edit script/cmtool.bat:

```
REM if not defined CHEF_URL set CHEF_64_URL=https://packages.chef.io/stable/windows/2008r2/chef-client-%CM_VERSION%-x64.msi
REM if not defined CHEF_URL set CHEF_32_URL=https://packages.chef.io/stable/windows/2008r2/chef-client-%CM_VERSION%-x86.msi

REM if defined ProgramFiles(x86) (
REM   SET CHEF_URL=%CHEF_64_URL%
REM ) else (
REM   SET CHEF_URL=%CHEF_32_URL%
REM )

if not defined CHEF_URL set CHEF_URL=https://packages.chef.io/stable/windows/2008r2/chef-client-%CM_VERSION%-x86.msi
```

**Comment by daxgames** [comment]
Fixed by PR #86

**Comment by daxgames** [comment]
Fixed and future proofed by PR #187

Is there anybody out there?

**Comment by arizvisa** [comment]
This is actually fixed by PR #180. @daxgames, if you're still interested in getting that merged..do you mind re-basing it? Lmk, if it's too much trouble.

**Comment by daxgames** [comment]
I will look at this today

**Comment by daxgames** [comment]
@arizvisa #180 is rebased.  It rebased cleanly and I have not tested it since rebasing.

I am not actively using it at this point but glad to see your involvement and passion.

**Comment by arizvisa** [comment]
Thx man. I'll work on integrating it.

**Comment by arizvisa** [comment]
This issue has been fixed by PR #200. If chef installations don't download and install due to a url issue, lmk and I'll re-open this.
